Formula/sdl2_sound.rb

```diff
@@ -0,0 +1,149 @@
+class Sdl2Sound < Formula
+  desc "Abstract soundfile decoder for SDL"
+  homepage "https://icculus.org/SDL_sound/"
+  # Includes fixes for CMake, just ahead of the release tag
+  url "https://github.com/icculus/SDL_sound/archive/06c8946983ca9b9ed084648f417f60f21c0697f1.tar.gz"
+  version "2.0.1"
+  sha256 "41f4d779192dea82086c8da8b8cbd47ba99b52cd45fdf39c96b63f75f51293e1"
+  license "Zlib"
+  head "https://github.com/icculus/SDL_sound.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "libmodplug"
+  depends_on "sdl2"
+  depends_on "timidity"
+
+  def install
+    args = std_cmake_args
+    args += [
+      "-DCMAKE_INSTALL_RPATH=#{rpath}",
+      "-DSDLSOUND_DECODER_WAV=TRUE",
+      "-DSDLSOUND_DECODER_AIFF=TRUE",
+      "-DSDLSOUND_DECODER_AU=TRUE",
+      "-DSDLSOUND_DECODER_VOC=TRUE",
+      "-DSDLSOUND_DECODER_FLAC=TRUE",
+      "-DSDLSOUND_DECODER_VORBIS=TRUE",
+      "-DSDLSOUND_DECODER_RAW=TRUE",
+      "-DSDLSOUND_DECODER_SHN=TRUE",
+      "-DSDLSOUND_DECODER_MODPLUG=TRUE",
+      "-DSDLSOUND_DECODER_MP3=TRUE",
+      "-DSDLSOUND_DECODER_MIDI=TRUE",
+    ]
+    args += "-DSDLSOUND_DECODER_COREAUDIO=TRUE" if OS.mac?
```

Comment:
```suggestion
    args << "-DSDLSOUND_DECODER_COREAUDIO=TRUE" if OS.mac?
```
`+=` only works for arrays here.